audit_log/middleware.py

```diff
@@ -33,6 +33,19 @@ def _get_user_from_request(request):
     return None
 
 
+def _get_fields_of_sender(sender, registry):
+        fields = []
+        if sender in registry:
+            fields = registry.get_fields(sender)
+        else:
+            # If sender model is not found in registry.
+            # Check it's base classes.
+            for base in sender.__bases__:
+                if base in registry:
+                    fields = registry.get_fields(base)
```

Comment:
Shouldn't we break the for-loop here?
Or we want to get the last possible base?

Reply:
I checked in a test project. Yes we will break there. I will create another commit